_ci/static_analysis.sh

```diff
@@ -13,11 +13,16 @@ java -jar vosp-api-wrappers-java-$VERACODE_WRAPPER_VERSION.jar -vid $VERACODE_AP
      ${RUN_IN_SANDBOX} -createprofile false \
      -filepath \
      alfresco-transformer-base/target/alfresco-transformer-base-*.jar \
-     alfresco-docker-alfresco-pdf-renderer/target/alfresco-docker-alfresco-pdf-renderer-*.jar \
-     alfresco-docker-imagemagick/target/alfresco-docker-imagemagick-*.jar \
-     alfresco-docker-libreoffice/target/alfresco-docker-libreoffice-*.jar \
-     alfresco-docker-tika/target/alfresco-docker-tika-*.jar \
-     alfresco-docker-transform-misc/target/alfresco-docker-transform-misc-*.jar \
+     alfresco-transform-pdf-renderer/alfresco-transform-pdf-renderer/target/alfresco-transform-pdf-renderer-*.jar \
+     alfresco-transform-pdf-renderer/alfresco-transform-pdf-renderer-boot/target/alfresco-transform-pdf-renderer-*.jar \
+     alfresco-transform-imagemagick/alfresco-transform-imagemagick/target/alfresco-transform-imagemagick-*.jar \
+     alfresco-transform-imagemagick/alfresco-transform-imagemagick-spring/target/alfresco-transform-imagemagick-spring-*.jar \
+     alfresco-transform-libreoffice/alfresco-transform-libreoffice/target/alfresco-transform-libreoffice-*.jar \
+     alfresco-transform-libreoffice/alfresco-transform-libreoffice-spring/target/alfresco-transform-libreoffice-spring-*.jar \
+     alfresco-transform-tika/alfresco-transform-tika/target/alfresco-transform-tika-*.jar \
+     alfresco-transform-tika/alfresco-transform-tika-spring/target/alfresco-transform-tika-spring-*.jar \
+     alfresco-transform-misc/alfresco-transform-misc/target/alfresco-transform-misc-*.jar \
+     alfresco-transform-misc/alfresco-transform-misc-spring/target/alfresco-transform-misc-spring*.jar \
```

Comment:
Should we decide on either "-boot" (or "-spring") for consistency ?

Reply:
Renamed to "-boot"